content/docs/latest/installing/community-platforms/scaleway.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Running Flatcar on Scaleway
+linktitle: Running on Scaleway
+weight: 10
+---
+
+These instructions will walk you through using Flatcar on [Scaleway][scaleway], importing an image, and running your first server using the command line interface. Please note that Scaleway is a community supported platform at the moment which means that the CI does not run on Scaleway images.
+
+## Import the image
+
+_Note_: This getting started assumes that Scaleway and AWS CLI are already installed and properly configured.
+
+It is possible to import your own Flatcar image in Scaleway using the [CLI][cli]. The process is done in two steps:
+1. Upload Flatcar image on Scaleway S3 bucket
+2. Create a snapshot from the S3 object
+
+For example, to upload an image from the Flatcar Beta channel:
+
+```bash
+$ wget https://beta.release.flatcar-linux.net/amd64-usr/current/flatcar_production_scaleway_image.img.bz2
+$ bzip2 --decompress ./flatcar_production_scaleway_image.img.bz2
+$ mv flatcar_production_scaleway_image.{img,qcow2}
+$ aws s3api create-bucket --bucket flatcar
+$ aws s3 cp flatcar_production_scaleway_image.qcow2 s3://flatcar/flatcar_production_scaleway_image.qcow2 
```

Comment:
Is this renaming to qcow2 necessary? If so, maybe we should think about creating scaleway images with qcow2.bz2 extension instead of img.bz2?